compiler/rustc_codegen_ssa/src/mir/operand.rs

```diff
@@ -132,7 +132,7 @@ impl<'a, 'tcx, V: CodegenObject> OperandRef<'tcx, V> {
         offset: Size,
     ) -> Self {
         let alloc_align = alloc.inner().align;
-        assert_eq!(alloc_align, layout.align.abi);
+        assert!(alloc_align >= layout.align.abi);
```

Comment:
Would it be more precise to say that `alloc_align` must be a multiple of `layout.align.abi`?

Reply:
Both are instances of `Align`, which only stores the exponent of the alignment. So the inequality does what we want.